apps/admin/frontend/src/screens/contest_adjudication_screen.tsx

```diff
@@ -337,13 +337,13 @@ export function ContestAdjudicationScreen(): JSX.Element {
     currentCvrId ? { cvrId: currentCvrId } : undefined
   );
   const voteAdjudicationsQuery = getVoteAdjudications.useQuery(
-    currentCvrId ? { cvrId: currentCvrId, contestId } : undefined
+    currentCvrId ? { cvrId: currentCvrId, contestId: contest.id } : undefined
```

Comment:
Is this intentional?

Reply:
Yes, because at this point TypeScript sees `contestId` as `ContestId | undefined`. We could keep this and add an `assert`.